cfg/builder/builder_finalize.cc

```diff
@@ -241,7 +241,9 @@ void CFGBuilder::markLoopHeaders(core::Context ctx, CFG &cfg) {
         }
     }
 }
-void CFGBuilder::removeDeadAssigns(core::Context ctx, const CFG::ReadsAndWrites &RnW, CFG &cfg) {
+void CFGBuilder::removeDeadAssigns(core::Context ctx, const CFG::ReadsAndWrites &RnW, CFG &cfg,
+                                   vector<UIntSet> &&blockArgs) {
```

Comment:
This particular bit seems like unusual C++: `removeDeadAssigns` doesn't actually do anything with the knowledge that the incoming value can be moved from.  Perhaps make this `const vector<UIntSet> &blockArgs` and then the caller can just declare the result of `fillInBlockArguments` as `const` to indicate that nothing is going to change the vector anywhere in the caller?

Reply:
Good idea; I can make that change post-review.